driver/src/main/java/org/neo4j/driver/internal/pool/StandardConnectionPool.java

```diff
@@ -71,25 +65,19 @@
      */
     private final ValidationStrategy<PooledConnection> connectionValidation;
 
-    private final int connectionsPerDatabase;
     private final Clock clock;
+    private final Config config;
 
-    public StandardConnectionPool( Logging logging )
-    {
-        this( loadConnectors( logging ) );
-    }
-
-    public StandardConnectionPool( Collection<Connector> conns )
+    public StandardConnectionPool( Config config )
```

Comment:
I kind of disagree of passing real parameters instead of Config. Because how we use Config is something like:
```
ClassA
{
  String param1;
  ClassB b;
  public ClassA( Config config )
  {
    param1 = config.param1;
    classB = new ClassB( config )
  }
}
ClassB 
{
  String param2;
  public ClassB( Config config )
  {
    param2 = config.param2;
  }
}
```